### PR TITLE
feat: [ZNO-2078] add custom plugin to provide ukrainian region map for superset

### PR DESCRIPTION
**Pull request by YuriiBorovenskyi**
**Description:**
* add custom plugin to provide Ukrainian region map for superset
* add ability to build map of Ukraine by name of regions
* add Ukrainian translations - po, json

**YouTrack:** https://youtrack.raccoongang.com/issue/ZNO-2078

**Merge checklist:**
* [ ] All reviewers approved
* [x] Demo status: OK
* [ ] Commits are squashed

### CODE REVIEW

File: superset/viz.py [hunk 1]
```diff
@@ -1544,12 +1544,59 @@ def query_obj(self) -> QueryObjectDict:
...
+                country=self.form_data["select_country"],
+                region_name=row["country_id"],
+            )
+        return data
+
+
+class RegionMapViz(BaseViz):
+
+    """A region centric"""
+
```
Comment (NikolayBorovenskiy) [inline superset/viz.py:1569]: Please fix this doc string.

**Review by NikolayBorovenskiy — CHANGES_REQUESTED** [review]
One tiny fix and let's go to check it out.

**Comment by NikolayBorovenskiy** [comment]
@YuriiBorovenskyi Pay attention to CI. It is hardly broken now.

File: superset/viz.py (hunk 1 above)
Comment (YuriiBorovenskyi) [reply superset/viz.py]: Done.

**Pull request merged**